README.md

```diff
@@ -32,16 +32,16 @@
 
 ### Project Setup
 
-- If you so choose you may install `create-react-app` globally by running `yarn global add create-react-app`.
-  - If you have CRA installed then run `create-react-app instagram` to create your starter application with the name `instagram`.
-  - If you don't have CRA installed then run `npx create-react-app instagram` to create your starter application with the name `instagram`.
-- Create a `components` directory inside the `src` directory, and then create a sub-directory called `SearchBar`, another one called `PostContainer`, and lastly one called `CommentSection`, all inside the `src` directory. Each of these directories should contain the component file as well as the CSS for their respective components. You'll also want to put any other components that coincide with your respective container components inside of these directories.
+[x] If you so choose you may install `create-react-app` globally by running `yarn global add create-react-app`.
+  [x] If you have CRA installed then run `create-react-app instagram` to create your starter application with the name `instagram`.
+  [x] If you don't have CRA installed then run `npx create-react-app instagram` to create your starter application with the name `instagram`.
+[x] Create a `components` directory inside the `src` directory, and then create a sub-directory called `SearchBar`, another one called `PostContainer`, and lastly one called `CommentSection`, all inside the `src` directory. Each of these directories should contain the component file as well as the CSS for their respective components. You'll also want to put any other components that coincide with your respective container components inside of these directories.
 
 ### Tasks (Day I)
 
-- There are three major container components that you'll need to implement for this project: the Search Bar, the Post Container, and the Comment Section.
+[x] There are three major container components that you'll need to implement for this project: the Search Bar, the Post Container, and the Comment Section.
 - At the end of Day I there will be a single instance of the Search Bar being rendered at the top of the page, as well as a Post Container and a Comment Section for every piece of mock data in the `dummy-data.js` file.
-- The root App component of your application should import the dummy data from the `dummy-data.js` file with `import dummyData from './dummy-data';` and iterate over said data, passing each individual object as a prop to an instance of `PostContainer`.
+[x] The root App component of your application should import the dummy data from the `dummy-data.js` file with `import dummyData from './dummy-data';` and iterate over said data, passing each individual object as a prop to an instance of `PostContainer`.
```

Comment:
consider keeping tack of these tasks in your PR comments instead of the readme 🤔 💭